lib/modules/hosts/pornbot.js

```diff
@@ -0,0 +1,89 @@
+//Borrowed heavily from gfycat.js
+modules['showImages'].siteModules['pornbot'] = {
+	domains: [ 'pornbot.net' ],
+	detect: function(href, elem) {
+		return (/^https?:\/\/(?:w{3}\.)?(?:v\.)?pornbot\.net\/[a-z0-9]{8,}$/i).test(href);
+	},
+	handleLink: function(elem) {
+		var hashRe = /^https?:\/\/(?:w{3}\.)?(?:v\.)?pornbot\.net\/[a-z0-9]{8,}$/i,
+			def = $.Deferred(),
+			groups = hashRe.exec(elem.href);
+
+		if (groups) {
+			var apiURL = location.protocol + '//pornbot.net/ajax/info.php?v=' + encodeURIComponent(groups[2]);
```

Comment:
Should be `groups[1]` since I only added one capturing group in https://github.com/honestbleeps/Reddit-Enhancement-Suite/pull/2272/files#r36432145